modules/oxide/src/less/theme/components/suggestededits/suggestededits.less

```diff
@@ -0,0 +1,567 @@
+//
+// Suggested Edits
+//
+@suggestededits-added-color: #78AB46;
+@suggestededits-modified-color: #006CE7;
+@suggestededits-removed-color: #CC0000;
+
+@suggestededits-color-tint: #006ce7;
```

Comment:
All these constants needs to be computed by other values I think that is the intent of the skining.